server/src/views/frame.js

```diff
@@ -241,8 +242,11 @@ class Frame extends React.Component {
           { nextClipNode }
         </div>
         <div id="full-page-button">
-          <img src={ this.props.staticLink("img/full-page-button.png") }
-            onClick={ this.clickFullPageButton.bind(this) } />
+          <a href="#" className="full-page-button-styles" onClick={ this.clickFullPageButton.bind(this) }>
```

Comment:
It occurs to me this could actually be `<a href="#frame">` or something similar to that, with no Javascript.